ignition-math/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Mykola Dolhyi <0xb000@gmail.com>
 pkgname=ignition-math
-pkgver=6.4.0
+pkgver=7.0.0
 pkgrel=3
```

Comment:
Since we are updating the PKGBUILD, the .SRCINFO also needs to be updated see: https://wiki.archlinux.org/index.php/.SRCINFO
This is also so that the people that are using this package know that the package has been updated.